src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -676,6 +676,7 @@ export class AppComponent implements OnInit, AfterViewChecked {
                     self.placeHolderDataSets[i].resized.emit();
                 }
             } else {
+                self.scrollEnabled = true;
```

Comment:
I don't understand why this fixes the problem.  Could you please explain a bit?  Thanks!

Reply:
When the number of result sets is less than the threshold, we were setting scrollEnabled to false here - https://github.com/microsoft/vscode-mssql/pull/1494/files/013d5f97dbfcfa1125084595dd898d1f88b5938b#diff-1df1f9bd5f3b0cf63b472db70bd235bfR673.

However, we never set it to true ever again, which was a bug. This needs to be true here - https://github.com/microsoft/vscode-mssql/pull/1494/files/013d5f97dbfcfa1125084595dd898d1f88b5938b#diff-1df1f9bd5f3b0cf63b472db70bd235bfR50

for the resultsets to emit the resize event.